src/com/google/javascript/jscomp/FindModuleDependencies.java

```diff
@@ -97,6 +98,17 @@ public void visit(NodeTraversal t, Node n, Node parent) {
 
     if (supportsEs6Modules && n.isExport()) {
       moduleType = ModuleType.ES6;
+      if (n.getBooleanProp(Node.EXPORT_DEFAULT)) {
+        // export default
+      } else if (n.getBooleanProp(Node.EXPORT_ALL_FROM)) {
+        // export * from 'moduleIdentifier';
+        // not yet supported by closure ES6 transpilation: 'Wildcard export' is not yet implemented.
+      } else if (n.hasTwoChildren()) {
+        // export {x, y as z} from 'moduleIdentifier';
+        Node moduleIdentifier = n.getLastChild();
+        Node importNode = IR.importNode(IR.empty(), IR.empty(), moduleIdentifier.cloneNode());
+        visit(t, importNode, parent);
```

Comment:
Rather than cloning a node and revisiting, can you refactor out the ES6 import case to a function `getModuleNameFromImportPath`? Then you can just call that with the 2nd child and add the result to the graph.